### PR TITLE
Minor fixes for upgrade downgrade tests to fix test failures

### PR DESCRIPTION
**Pull request by su225**
To fix a test failure - https://storage.googleapis.com/istio-prow/logs/istio-downgrade-using-istioctl-1.8_latest-1.7_latest/1333984254945660928/build-log.txt

### CODE REVIEW

**Review by carolynhu — COMMENTED** [review]
@su225 did you test this change? do we really need that much long for the timeout?

**Comment by su225** [comment]
@carolynhu - Sorry. I used the same timeout all over the place as these tests are slow. I found out that the job runs for 500s. So it is enough to wait for slightly above 10 minutes to take into account all the things that happen before and after running the pod in Kubernetes (in my local environment, I had run into pathetic issues. So I have set it to 12 minutes or 720s to be on safe side)

File: upgrade_downgrade/test_dual_control_plane_upgrade_downgrade_boutique.sh [hunk 1]
```diff
@@ -123,8 +123,11 @@ kubectl label namespace "${TEST_NAMESPACE}" istio-injection- || echo "istio-inje
 kubectl label namespace "${TEST_NAMESPACE}" istio.io/rev="${FROM_REVISION}"
 
 function deploy_and_wait_for_services() {
-  local -n services="$1"
-  for svc in "${services[@]}"; do
+  # Do not name this services. We'll end up with a situation
```
Comment (carolynhu) [inline upgrade_downgrade/test_dual_control_plane_upgrade_downgrade_boutique.sh:126]: nit: Add `Warning: ` in the beginning of this comments

**Comment by carolynhu** [comment]
LGTM, only one comment.

File: upgrade_downgrade/test_dual_control_plane_upgrade_downgrade_boutique.sh (hunk 1 above)
Comment (su225) [reply upgrade_downgrade/test_dual_control_plane_upgrade_downgrade_boutique.sh]: Done. Added the warning

Actually it is easy to run into this issue. It was caused by this
```bash
function deploy_and_wait_for_services() {
    local -n services="$1"
    # ... rest of the stuffs
}
# Calling with services as parameter expanded to
# local -n services=services and caused the error
deploy_and_wait_for_services services
```

**Review by carolynhu — APPROVED** [review]

**Pull request merged**